src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -3596,10 +3601,6 @@ impl Coordinator {
                 )
             };
 
-            // Timestamp selection
-            let since = self.least_valid_read(&global_mir_plan.id_bundle());
-            let global_mir_plan = global_mir_plan.resolve(since);
-
```

Comment:
This is assuming that we don't need to select a timestamp when explaining indexes or MVs. If that's not correct, we can re-introduce the timestamp selection below after the LIR plan was created.

Reply:
We don't at the moment, and since it simplifies code I think we should do that until somebody requests that.